src/ripple/app/hook/applyHook.h

```diff
@@ -1380,7 +1381,15 @@ class HookExecutorJS : public HookExecutorBase
         JLOG(j.trace()) << "HookInfo[" << HC_ACC()
                         << "]: creating quickjs instance";
 
-        QuickJSVM vm{reinterpret_cast<void*>(&hookCtx), instructionLimit};
+        uint64_t dateNow =
+            std::chrono::duration_cast<std::chrono::milliseconds>(
+                hookCtx.applyCtx.view()
+                    .info()
+                    .parentCloseTime.time_since_epoch())
+                .count() +
+            946684800000;
```

Comment:
Some constant would be nice

Reply:
Yes, I considered that.
At the moment, the offset constant has not been introduced in rippled1.11.0 which is the basis for xahaud, so it is done like this.

https://github.com/XRPLF/rippled/pull/4775